resources/nats/templates/_helpers.tpl

```diff
@@ -47,10 +47,6 @@ Selector labels
 {{- define "nats.selectorLabels" -}}
 {{- if .Values.nats.selectorLabels -}}
 {{ .Values.nats.selectorLabels | toYaml }}
-{{- else if eq .Release.Name "eventing-nats" -}}
```

Comment:
Why are you removing this? It is needed for upgrade from kyma to nats-manager!!!

Reply:
Can you please explain me why do you need this for upgrade? 
The reason why I did it is that I want to have a different instance label for each nats instance. I retrieve & delete PVCs with a unique label.

Reply:
You can get the label from statefulset manifest which you get in handleDelete method. 
This part is needed for upgrade because the existing labels in Kyma do not match with what we generate here.

Reply:
what do you think about the [logic](https://github.com/muralov/nats-manager/blob/62059de98eddb7bc2b53c4c34f57630c64675832/internal/controller/nats/deprovisioner.go#L68C9-L68C9) I applied? It is easy than sts and we'll delete it after modular Kyma migration. Moreover, sts has the `-nats`suffix added in the integration tests (different from CR name), which made it difficult to implement with sts.